Formula/jfrog-cli.rb

```diff
@@ -17,14 +18,17 @@ class JfrogCli < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", "-ldflags", "-s -w -extldflags '-static'", "-trimpath", "-o", bin/"jfrog"
+    system "go", "build", *std_go_args(ldflags: "-s -w -extldflags '-static'", output: bin/"jf")
     prefix.install_metafiles
+    bin.install_symlink "jf" => "jfrog"
+
     system "go", "generate", "./completion/shells/..."
     bash_completion.install "completion/shells/bash/jfrog"
-    zsh_completion.install "completion/shells/zsh/jfrog" => "_jfrog"
+    zsh_completion.install "completion/shells/zsh/jfrog"
   end
 
   test do
```

Comment:
Can you also think of a more substantial test than just checking the version? That doesn't generally reveal any issues.

Reply:
We already have a very large coverage https://github.com/jfrog/jfrog-cli/actions/runs/1790209337.
In this case, Homebrew only downloads and installs the JFrog CLI. I can't think about anything else, but I'm open for new ideas.

Reply:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I added a ping command to a public Artifactory server.